estate_account/models/estate_property.py

```diff
@@ -16,7 +18,10 @@ def action_set_property_as_sold(self):
             "invoice_line_ids": invoice_line_vals,
         }
 
+        # self.env["account.move"].check_access_rights("write")
+        # print(" reached ".center(100, '='))
```

Comment:
Use comments only to document code.

Reply:
I would like to keep in my training project as reference to know how to check access rights in code. May I?

Reply:
oh you can save it in a gist